### ISSUE TITLE
큰 프로젝트에서의 문제

### ISSUE THREAD

**Issue by zepinos** [naver/yobi#909]
예전에 단일 파일의 용량이 클 경우 커밋이 실패한다는 글을 올렸는데, 오늘 0.8.0 올리다가 데이터가 맛이 가서 새로 import 부터 하고 있는데, 여전히 데이터 크기 때문인지 메모리 오류로 자료들이 다 안올라가고...문제가 많이 보이네요.

svn 은 별도로 svnserv 로 열어서 svn:// 프로토콜로 주고받아도 되는지 궁금합니다.

**Comment by npcode** [comment]
네 Git과 달리 Yobi가 svn 저장소에는 hook을 걸지도 않으니 직접 svnserve 띄워 쓰셔도 문제 없을 것입니다.

**Comment by npcode** [comment]
큰 파일의 크기는 어느정도인가요? 그리고 Yobi에 메모리는 얼마나 할당해주셨나요?

**Comment by zepinos** [comment]
xms 256m, xmx 4096m 줬습니다. 전체 디렉토리는 26기가, 게임 디자이너들(원화 등)의 포토샵 이미지들이 파일당 1GB 이상 되는 것들이 다수입니다.

**Comment by zepinos** [comment]
받는거야 그런데...올리는건 인증 정보를 파일 방식으로 하질 않으니...이것도 참 난감하네요. 사용자 비밀번호를 평문으로 입력하는게 싫어서 요비 쓰는건데...

**Comment by npcode** [comment]
제가 실험을 해 보니, 메모리 옵션 주지 않고 실행했을 때, 1.5GB 파일 한개는 문제없이 커밋되고 2GB 파일은 PlayFramework 제약때문에(요청 payload가 2GB를 넘을 수 없음) Request Entity Too Large 에러가 나는군요.

문제가 발생하는 파일의 크기가 어느정도인지요?
